code/datums/perks/fate.dm

```diff
@@ -0,0 +1,197 @@
+		var/datum/reagent/drug = new drugtype
+		holder.metabolism_effects.addiction_list.Add(drug)
+		var/obj/item/weapon/storage/pill_bottle/PB = new /obj/item/weapon/storage/pill_bottle(T)
+		PB.name = "bottle of happiness"
+		for(var/i=1 to 7)
+			var/obj/item/weapon/reagent_containers/pill/pill = new /obj/item/weapon/reagent_containers/pill(T)
+			pill.reagents.add_reagent(drug.id, pill.volume)
+			pill.name = "happy pill"
+			PB.handle_item_insertion(pill)
+		holder.put_in_hands(PB)
+
+/datum/perk/alcoholic
+	name = "Alcoholic"
+	icon_state = "beer" //https://game-icons.net/1x1/delapouite/beer-bottle.html
+	desc = "For you it's alcohol: the cause of, and solution to, all of life's problems."
+
+/datum/perk/alcoholic/assign(mob/living/carbon/human/H)
+	..()
+	var/ethanoltype = pick(subtypesof(/datum/reagent/ethanol))
+	if(!(ethanoltype in holder.metabolism_effects.addiction_list))
+		var/datum/reagent/alcohol = new ethanoltype
+		holder.metabolism_effects.addiction_list.Add(alcohol)
+
+/datum/perk/alcoholic_active
+	name = "Alcoholic (active)"
+	icon_state = "drinking" //https://game-icons.net/1x1/delapouite/drinking.html
+
+/datum/perk/alcoholic_active/assign(mob/living/carbon/human/H)
+	..()
+	holder.stats.addTempStat(STAT_ROB, 10, INFINITY, "Fate Alcoholic")
+
+/datum/perk/alcoholic_active/remove()
+	holder.stats.addTempStat(STAT_ROB, "Fate Alcoholic")
+	..()
+
+/datum/perk/noble
+	name = "Noble"
+	icon_state = "family" //https://game-icons.net
+	desc = "You're a noble, a graceful ornament to the civil order. The jewel of society. Waiting to be stolen."
+
+/datum/perk/noble/assign(mob/living/carbon/human/H)
+	..()
+	if(!holder.last_name)
+		qdel(src)
+	holder.sanity.environment_cap -= 1
+	var/turf/T = get_turf(holder)
+	var/obj/item/W = pickweight(list(/obj/item/weapon/tool/knife/butterfly = 1,
+				/obj/item/weapon/tool/knife/switchblade = 1,
+				/obj/item/weapon/tool/knife = 1,
+				/obj/item/weapon/tool/knife/boot = 0.5,
+				/obj/item/weapon/tool/knife/hook = 2,
+				/obj/item/weapon/tool/knife/ritual = 0.5,
+				/obj/item/weapon/tool/scythe = 0.3,
+				/obj/item/weapon/tool/sword = 0.2,
+				/obj/item/weapon/tool/sword/katana = 0.2,
+				/obj/item/weapon/tool/knife/butch = 2,
+				/obj/item/weapon/tool/knife/dagger = 0.8))
+	W = new W(T)
+	W.name = "[holder.last_name] family [W.name]"
+	holder.put_in_hands(W)
+
+/datum/perk/noble/remove()
+	holder.sanity.environment_cap += 1
+	..()
+
+/datum/perk/rat
+	name = "Rat"
+	desc = "You have been always clever, but it is one thing to be clever and another to be wise."
+	icon_state = "rat"
+
+/datum/perk/rat/assign(mob/living/carbon/human/H)
+	..()
+	holder.sanity.max_level -= 10
+
+/datum/perk/rat/remove()
+	holder.sanity.max_level += 10
+	..()
+
+/datum/perk/rejected_genius
+	name = "Rejected genius"
+	icon_state = "knowledge"
+	desc = "Your dreams are undisturbed by reality and you are in search of the philosopher’s stone and the elixir of life."
```

Comment:
"Your dreams are undisturbed by reality, your search for the impossible continues regardless of your peers."